components/code-block.tsx

```
@@ -47,11 +51,11 @@ const CodeBlock = async ({
         )}
         data-language={lang || undefined}
         data-line-numbers={showLineNumbers || undefined}
-        dangerouslySetInnerHTML={{ __html: codeHighlighted }}
+        dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)